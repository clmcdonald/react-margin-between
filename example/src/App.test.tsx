import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import { render } from '@testing-library/react';

describe('example tests', () => {
  test('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<App />, div)
    ReactDOM.unmountComponentAtNode(div)
  });

  test('matches snapshot', () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});
