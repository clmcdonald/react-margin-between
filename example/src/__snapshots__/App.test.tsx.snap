// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example tests matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvtL fYAlYH"
  >
    <div
      class="sc-gsDKAQ bHBcAv"
    >
      <div
        class="sc-hKwDye jNowCY"
        style="margin-right: 100px;"
      >
        Child 1
      </div>
      <div
        class="sc-hKwDye jNowCY"
        style="margin-right: 100px; margin-left: 100px;"
      >
        Child 2
      </div>
      <div
        class="sc-hKwDye jNowCY"
        style="margin-left: 100px;"
      >
        Child 3
      </div>
    </div>
    <br />
    <div
      class="sc-dkPtRN iEqVaT"
    >
      <div
        class="sc-hKwDye jNowCY"
        style="margin-bottom: 30px;"
      >
        Child 1
      </div>
      <div
        class="sc-hKwDye jNowCY"
        style="margin-bottom: 30px; margin-top: 30px;"
      >
        Child 2
      </div>
      <div
        class="sc-hKwDye jNowCY"
        style="margin-top: 30px;"
      >
        Child 3
      </div>
    </div>
  </div>
</DocumentFragment>
`;
